src/dao/services/types.ts

```diff
@@ -87,6 +87,7 @@ export interface GovernanceDAODetails {
   votingDelay: number;
   votingPeriod: number;
   minimumProposalDeposit?: number;
+  lockedTokenDetails?: any[];
 }
```

Comment:
we can define type here instead of `any`

Reply:
Done